site/content/resources.md

```diff
@@ -96,6 +97,13 @@ This program provides primary care for uninsured patients, including immigrants
 
 To find local community health centers, search the Community Clinic Association of Los Angeles County.
 
+### Latino Health Resources
```

Comment:
Could you add a new line after the header to maintain consistency in terms of formatting?

Reply:
Should be all set!

I'll look around for some good layouts for resource pages, I'll keep the conversation going in #13